cloudigrade/api/tests/views/test_dailyconcurrentusageviewset.py

```diff
@@ -1,229 +1,25 @@
-            body["start_date"], [_("start_date must be a date (YYYY-MM-DD).")]
-        )
-        self.assertEqual(body["end_date"], [_("end_date must be a date (YYYY-MM-DD).")])
-
-    def test_start_date_is_inclusive_and_end_date_is_exclusive(self):
-        """Test that start_date is inclusive and end_date is exclusive."""
-        start_date = datetime.date(2019, 1, 1)
-        end_date = datetime.date(2019, 1, 4)
-        data = {
-            "start_date": start_date.strftime("%Y-%m-%d"),
-            "end_date": end_date.strftime("%Y-%m-%d"),
-        }
-        api_helper.calculate_concurrent(start_date, end_date, self.user1.id)
-
-        client = APIClient()
-        client.force_authenticate(user=self.user1)
-        response = client.get(
-            "/api/cloudigrade/v2/concurrent/", data=data, format="json"
-        )
-        body = response.json()
-        self.assertEqual(body["meta"]["count"], 3)
-        self.assertEqual(len(body["data"]), 3)
-        self.assertEqual(body["data"][0]["date"], "2019-01-01")
-        self.assertEqual(body["data"][1]["date"], "2019-01-02")
-        self.assertEqual(body["data"][2]["date"], "2019-01-03")
-
-    def test_start_date_end_date_defaults(self):
-        """
-        Test with no start_date and no end_date set.
-
-        Default start_date is "today" and default end_date is "tomorrow", and
-        since start_date is inclusive and end_date is exclusive, the resulting
-        output should be data for one day: today.
-        """
-        today = get_today()
-        data = {}
-        api_helper.calculate_concurrent(
-            today, today + datetime.timedelta(days=1), self.user1.id
-        )
-
-        client = APIClient()
-        client.force_authenticate(user=self.user1)
-        response = client.get(
-            "/api/cloudigrade/v2/concurrent/", data=data, format="json"
-        )
-        body = response.json()
-        self.assertEqual(body["meta"]["count"], 1)
-        self.assertEqual(len(body["data"]), 1)
-        self.assertEqual(body["data"][0]["date"], str(today))
-
-    def test_future_end_date_returns_400(self):
-        """
-        Test with far-future end_date, expecting it to return 400.
-
-        When an end_date is given that is later than tomorrow, we return
-        a 400 response, because we cannot predict the future.
-        """
-        today = get_today()
-        future = today + datetime.timedelta(days=100)
-        data = {"end_date": str(future)}
-        api_helper.calculate_concurrent(today, future, self.user1.id)
-
-        client = APIClient()
-        client.force_authenticate(user=self.user1)
-        response = client.get(
-            "/api/cloudigrade/v2/concurrent/", data=data, format="json"
-        )
-        self.assertEqual(response.status_code, 400)
-
-        body = response.json()
-        self.assertEqual(body["end_date"], [_("end_date cannot be in the future.")])
+        SharedDailyConcurrentUsageViewSetTest.setUp(self)
```

Comment:
The proper convention in Python for calling the parent class is to use `super` like this instead of directly referencing it:
```suggestion
        super(DailyConcurrentUsageViewSetTest, self).setUp()
```